posthog/temporal/data_imports/__init__.py

```diff
@@ -4,6 +4,7 @@
     update_external_data_job_model,
     run_external_data_job,
     validate_schema_activity,
+    create_source_templates,
```

Comment:
You had included this in the activities below but needed to import here too. I'm guessing the activities addition felt like an import but wasn't. those are used to define the tasks that get registered on temporal